user_docs/en/userGuide.t2t

```diff
@@ -2593,52 +2593,67 @@ For more information, please see the [NVDA Developer Guide https://www.nvaccess.
 
 ++ Add-on Store ++[AddonsManager]
 The Add-on Store allows you to browse and manage add-on packages for NVDA.
-These packages are provided by the community and contain custom code that may add or change features in NVDA or even provide support for extra Braille displays or speech synthesizers.
-The Add-on Store is accessed by the Tools submenu in the NVDA menu.
+Add-ons are provided by the community and contain custom code that may add or enhance support for certain applications, add or change features in NVDA, and provide support for extra Braille displays or speech synthesizers.
+While the portion of NVDA which manages add-ons is called the "Add-on Store", the vast majority of add-ons are free.
+The rare exceptions are usually add-ons used to provide support for certain commercial speech synthesizers, and any paid portion of these add-ons will not be available through the Add-on Store.

```

Comment:
It might be simpler to just state that add-ons distributed through the add-on store are free, and that paid external add-ons exist.
I think commenting on the % might be misleading, as certain commercial add-ons are popular.

```suggestion
Add-ons that are available in the Add-on Store are free.
Other add-ons which require purchasing exist, such as supporting commercial speech synthesizers, and can be installed from external sources.
```

Reply:
> I think commenting on the % might be misleading, as certain commercial add-ons are popular.

Out of curiosity, are you talking about where I wrote "the rare exception", or where I wrote "the vast majority are free"? I didn't think of either as commenting on the percent. As it stands, however, out of all 94 add-ons currently available in the largest list the store can generate, only one of them is a synth, and Its description says nothing about having a paid component.

The community site does have a couple of the Vocalizer add-ons, and I imagine those will eventually join the store. My assumption was that those two could be installed, but required a paid component to function, which is why I wrote it that way.

Note that I meant that the number of add-ons was low, not that the number of users for them was, which is how you appeared to understand it. Will amend.

Reply:
I was referring to both of those phrases, I was just hoping to simplify the language and make your points more direct.

I was assuming your intention behind these changes was the following:
- note that the add-ons available in the store are all free
- note that commercial/paid add-ons (or components to add-ons) exist
- note that paid add-ons are the minority

I figured the last point was not necessary, and might change in the future, as it's not really up to NV Access or NVDA.